src/core/public/chrome/chrome_service.tsx

```diff
@@ -465,6 +537,16 @@ export interface ChromeStart {
    */
   setIsVisible(isVisible: boolean): void;
 
+  /**
+   * Get an observable of the current header variant.
+   */
+  getHeaderVariant$(): Observable<HeaderVariant | undefined>;
```

Comment:
can it return page by default instead of being undefined?

Reply:
After talking to Viraj, I will revisit where the default is set to be consistent.

Reply:
The Header component currently owns all of the defaults:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/dcefaad483d63a7733e1612c2e94774a90fce23f/src/core/public/chrome/ui/header/header.tsx#L137-L143

Will leave it as is for consistency.